integration_tests/pom.xml

```diff
@@ -417,7 +417,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
-                <version>3.0.0</version>
+                <version>3.3.2</version>
```

Comment:
Not sure if this for this PR but since you are touching this:

It looks like there are only one minor customization to the javadoc plugin between the parent pom and this pom.
We could delete the plugin definition here and add a parameter for `detectOfflineLinks` with the default value `true`. And this pom will merely override the parameter.

Reply:
thanks! I will try simplify the javadoc plugin def later